integration-tests/cli/test/metadata.test.ts

```diff
@@ -126,3 +126,18 @@ test.serial('does not log credentials', async (t) => {
   t.regex(logString, /(pin_number)/i);
   t.notRegex(logString, new RegExp(sensitiveValue), 'i');
 });
+
+// Test for auto-installing adaptor
+test.serial(
+  `openfn metadata -f -S "${state}" -a common --log-json --log info`,
+  async (t) => {
+    const { stdout } = await run(t.title);
+
+    t.regex(stdout, /Generating metadata/);
+    t.regex(stdout, /Installing packages.../);
+    t.regex(stdout, /Installed @openfn\/language-common@1.13.1/);
```

Comment:
This will fail later because the version of common will change.

You either need to use a more forgiving regex, like `/Installed @openfn\/language-common@` (which is fine), or lock the version in the command above, ie by adding `-a common@1.31.1`

I'm happy with both solutions - perhaps you can pick your favourite and tell me why?

Reply:
Hey, I have decided to improve the regex syntax instead of specifying the version of the adaptor for the following reasons:
1. This approach is resilient to changes in the version of the common package. It looks for a general pattern rather than a specific version number, making it more future proof.
2. As new versions of the common package are released, as long as the installation message format remains consistent, the regex will still match.